readthedocs/gold/templates/gold/subscription_form.html

```diff
@@ -59,6 +59,9 @@ <h2>Read the Docs Gold</h2>
       we suggest giving at least $20/month to help cover our support and operations costs.
```

Comment:
I feel like these other parts of the page are just blabbing in an awkward way also. I'm almost tempted to just remove them?

Reply:
I'll give it a read and see if there's anything that should be removed. I tried to remove anything inaccurate already.

Reply:
I do think everything around [domains](https://github.com/davidfischer/readthedocs.org/blob/a466bf6385ffbd5f758c9f56b0fe9840816ae089/readthedocs/gold/templates/gold/subscription_form.html#L71-L90) can be deleted once we get Lets Encrypt support. I don't quite think I'll delete that yet.

Reply:
Why would LE support change our infra cost for domains? If anything, it would increase the overhead we have for each domain we serve, managing the certs, etc.